__tests__/lib/pg/service.test.js

```diff
@@ -29,11 +29,20 @@ const scpPostgresCredentials = {
   password: 'postgres',
 }
 const scpModel = './__tests__/__assets__/cap-proj/srv/'
-
+const localSchemaCredentials = {
+  hostname: 'localhost',
+  port: '5432',
+  dbname: 'beershop',
+  username: 'postgres',
+  password: 'postgres',
+  schema: 'superbeer',
+}
+const localSchemaModel = './__tests__/__assets__/cap-proj/srv/'
 // run test suite with different sets of data
 describe.each([
   ['local', localCredentials, localModel],
   ['scp', scpPostgresCredentials, scpModel],
+  ['local-with-schema', localSchemaCredentials, localSchemaModel],
```

Comment:
same as above: could you please integrate this into `__tests__/lib/pg/_buildSuiteEnvironments.js` ?